pkg/storage/minio/minio.go

```diff
@@ -12,6 +12,7 @@ import (
 
 type storageImpl struct {
 	minioClient *minio.Client
+	minioCore *minio.Core
```

Comment:
Is there a reason to switch from minio.Client to minio.Core when both of them can use ListObjectsV2?